setup.py

```diff
@@ -71,7 +71,11 @@ def run(self):
     install_requires.append("networkx>=2.1")
 
 # Extra dependencies for remote integrations
-gs = ["google-cloud-storage==1.13.0"]
+
+# Restricting google-resumable-media version is due to fact that
+# since 0.3.3 our builds for Linux and MacOs started to fail. This restriction
```

Comment:
```suggestion
# since 0.3.3 our binaries built by pyinstaller started to fail on Linux and macOS. This restriction will be removed after https://github.com/iterative/dvc/issues/2441 is resolved.
```